ee/codegen/src/__test__/nodes/inline-prompt-node.test.ts

```diff
@@ -513,4 +513,36 @@ describe("InlinePromptNode", () => {
       expect(await writer.toStringFormatted()).toMatchSnapshot();
     });
   });
+
+  describe("LEGACY prompt node API failure handling", () => {
```

Comment:
this test doesn't actually repro the test failure we saw on main. instead, create a test on project.test.ts that simply has a LEGACY variant and ensure that this node file's exist